homeassistant/components/ebusd/__init__.py

```diff
@@ -0,0 +1,131 @@
+"""
+Support for Ebusd daemon for communication with eBUS heating systems.
+
+For more details about ebusd deamon, please refer to the documentation at
+https://github.com/john30/ebusd
+"""
+
+from datetime import timedelta
+import logging
+import socket
+
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_NAME, CONF_HOST, CONF_PORT, CONF_MONITORED_CONDITIONS)
+from homeassistant.exceptions import PlatformNotReady
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.discovery import load_platform
+from homeassistant.util import Throttle
+
+from .const import (
+    DOMAIN, SENSOR_TYPES, READ_COMMAND, WRITE_COMMAND)
+
+REQUIREMENTS = ['ebusdpy==0.0.4']
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_EBUSD = 'EBUSD'
+DEFAULT_NAME = 'ebusd'
+DEFAULT_PORT = 8888
+CONF_CIRCUIT = 'circuit'
+CACHE_TTL = 900
+SERVICE_EBUSD_WRITE = 'ebusd_write'
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=15)
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_CIRCUIT): cv.string,
+        vol.Required(CONF_HOST): cv.string,
+        vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(CONF_MONITORED_CONDITIONS, default=[]): vol.All(
+            cv.ensure_list, [vol.In(SENSOR_TYPES['700'])])
+    })
+}, extra=vol.ALLOW_EXTRA)
+
+
+def setup(hass, config):
+    """Set up the ebusd component."""
+    if DOMAIN not in config:
+        return True
+
+    name = config[DOMAIN].get(CONF_NAME)
```

Comment:
> Since all of the config parameters have defaults, you don't need to use `.get` use the [] instead.

Which is the difference? Using get method is the same thing that access directly by string